charts/coredns/Chart.yaml

```diff
@@ -21,3 +21,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: fixed
       description: Fix HPA configuration
+    - kind: fixed
+      description: Fix assignment of NET_BIND_SERVICE capability, enable use of pullSecrets with Deployment
```

Comment:
This should completely replace the current change log, not append to it. Also, please break it down to 2 items. The final structure should look similar to:

```yaml
artifacthub.io/changes: |
  - kind: fixed
    description: Fix assignment of NET_BIND_SERVICE capability
  - kind: fixed
    description: Enable use of pullSecrets with Deployment
```